File: app/src/main/java/database/TopicDatabaseSchema.java
package database;

public class TopicDatabaseSchema {
    public static final class TopicTable {
        public static final String NAME = "topics";

        public static final class Cols {
            public static final String UUID = "uuid";
            public static final String TOPIC = "topic";
            public static final String VIEWS = "views";
            public static final String ENABLED = "enabled";
            public static final String TOPIC_SEARCHER = "topic_searcher";
            public static final String NOTIFIED_VIDEOS = "notified_videos";
            public static final String TOP_VIDEO_NOTIFICATION_SHOWN = "top_video";
        }
    }

}

File: app/src/main/java/database/TopicDatabaseSchema.kt
package database

class TopicDatabaseSchema {
    object TopicTable {
        const val NAME = "topics"

        object Cols {
            const val TOPIC_TYPE = "topic_type"
            const val UUID = "uuid"
            const val TOPIC = "topic"
            const val VIEWS = "views"
            const val ENABLED = "enabled"
            const val TOPIC_IDS = "topic_searcher"
            const val PREVIOUS_NOTIFICATIONS = "notified_videos"
            const val FIRST_NOTIFICATION_SHOWN = "top_video"
            const val RETWEETS = "retweets"
            const val TWEET_LIKES = "tweet_likes"
        }
    }

}
